website/cue/reference/components/sources/kubernetes_logs.cue

```diff
@@ -263,6 +263,15 @@ components: sources: kubernetes_logs: {
 				unit:    "milliseconds"
 			}
 		}
+		delay_deletion_ms: {
+			common:      false
+			description: "Delay between receiving a `DELETE` event and removing any related metadata Vector has stored. This controls how quickly Vector will remove metadata for resources that have been removed from Kubernetes, a longer delay will allow Vector to continue processing and enriching logs after the source Pod has been deleted."
```

Comment:
I might word wrap this 😄 

Just to clarify, what happens if the metadata is deleted before Vector has finished processing the log files for those pods?

Reply:
😆 I expected cue fmt to do so but.... I figured it would either wrap it or be angry about my manual wrapping - I can break it up. I'll update the description to mention what happens if things have been deleted.